src/navigation/drive.py

```diff
@@ -14,6 +16,7 @@ def get_drive_command(
     rover_pose: SE3,
     completion_thresh: float,
     turn_in_place_thresh: float,
+    drive_direction: int = 1,
```

Comment:
I think this value will be 1 or -1 depending on if the rover will go forwards/backwards. We use this for the j-turn specifically. Since we multiply our rover direction and linear x velocity by this number, a bool probably wouldn't make sense in this case.